### ISSUE TITLE
Feedback: 'NServiceBus Router two-way bridge topology'

### ISSUE THREAD

**Issue by nbarnwell** [Particular/docs.particular.net#5371]
Feedback for 'NServiceBus Router two-way bridge topology' https://docs.particular.net/nservicebus/router/bridge

Location in GitHub: https://github.com/Particular/docs.particular.net/blob/master/nservicebus/router/bridge.md

I was wondering if this page could include some guidance on what you expect in terms of the number of _instances_ of the router? The diagram here suggests that **all messages** from the various SQL endpoints on the left go through the same router instance to get to the RabbitMQ endpoints. This means the router instance is a single point of failure for potentially a lot of things.

Would you recommend running an instance of the router per logical endpoint?:

![image](https://user-images.githubusercontent.com/842125/122749912-cd31ac00-d285-11eb-8ece-bbf7ad0a04b8.png)

**Comment by SzymonPobiega** [comment]
Hi @nbarnwell. Thank for you feedback. The router can be scaled out as any NServiceBus endpoints by deploying multiple instances. 

On MSMQ it can be a bit more tricky than on other transports because of the local nature of MSMQ instances. You might consider deploying the Router to a fail-over cluster just like with the Distributor https://docs.particular.net/transports/msmq/distributor/deploying-to-a-cluster.

Bottom line is, you probably don't want to go through the complexity of setting up a router per each logical endpoint but rather have a single cluster of routers for a given pair of transports (e.g. MSMQ/ASB).

**Comment by SzymonPobiega** [comment]
I added a section on scaling out to the main router doco page https://docs.particular.net/nservicebus/router/#scaling-out
